### DISCUSSION TITLE
Coesistenza 2 hub zigbee

### DISCUSSION THREAD

**Discussion by marcoferroha** [indomus/forum#2005] [HUB: Altro (Homey, iHost, SmartThings, Fibaro ecc.)]
E' possibile far coesistere su HA due hub Zigbee in contemporanea ?
In particolare:
-- Conbeeii
--[ITEAD Sonoff ZigBee 3.0 USB Dongle Plus v2 (ZBDongle-E)

il tutto in ambiente Docker su Raspberry

**Comment by snakuzzo** [comment]
sì

**Reply by marcoferroha** [reply]
il problema è che ho configurato il deconz per il combeeii su /dev/ttyACM0 e funziona perfettamente, se inserisco l'altro hub senza ancora fare nulla a livello di configurazione non funziona più Combeeii come se ci fosse un conflitto di porte....
Hai qualche suggerimento?

**Reply by snakuzzo** [reply]
Sei stato poco chiaro.
Se inserisci l'altra chiavetta devi vedere qual è la porta corretta. 
Probabile sia `/dev/ttyACM1` o magari `/dev/ttyUSB0`...o quello che è.
In base a questo fai il bind del device sul container docker (immagino tu abbia capito come funziona)

**Comment by marcoferroha** [comment]
Buongiorno, 
ho ripreso dopo qualche anno un nuovo raspberry (5) e ho Conbee III ovviamente usb e ITEAD Sonoff ZigBee 3.0 USB Dongle Plus v2 (ZBDongle-E) sulle due porte usb 3.0 . Installando il Docker su raspbian os come faccio a farle la vorare insieme senza conflitti ?
Perdonatemi l'ignoranza ma non ne vengo a capo....
Grazie

**Reply by KC-inDomus** [reply]
scusa, perché dovresti usare due coordinator

**Reply by marcoferroha** [reply]
perchè mi ricordo che su HA c'era qualche dispositivo che era integrato meglio su uno piuttosto che sull'altro....

**Reply by KC-inDomus** [reply]
sei male informato. il coordinator hardware c'entra molto poco con la modalità di integrazione, a fare da spartiacque è il software che usi; metter su due reti zigbee non ha alcun senso.

**Reply by marcoferroha** [reply]
Grazie mille , farò tesoro del tuo consiglio
